home_works/lesson12/lsn12.py

```diff
@@ -0,0 +1,53 @@
+class Moving:
+    def move(self):
+        raise NotImplementedError()
+
+
+class Animal(Moving):
+    def voice(self):
+        raise NotImplementedError()
+
+
+class Transport(Moving):
+
+    def launch(self):
+        raise NotImplementedError()
+
+
+class Car(Transport):
+    def __init__(self, status='not_started'):
```

Comment:
remove your param.. we cant launch car without using launch method
But leave attribute with value